package.json

```diff
@@ -25,33 +25,33 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "10.0.0-rc.0",
-    "@angular/cdk": "10.0.0-next.1",
-    "@angular/cdk-experimental": "10.0.0-next.1",
-    "@angular/common": "^10.0.0-rc.0",
-    "@angular/compiler": "^10.0.0-rc.0",
+    "@angular/animations": "10.0.0",
+    "@angular/cdk": "10.0.0",
+    "@angular/cdk-experimental": "10.0.0",
+    "@angular/common": "^10.0.0",
+    "@angular/compiler": "^10.0.0",
     "@angular/components-examples": "angular/material2-docs-content#master",
```

Comment:
Should probably use `#10.0.x`?

Reply:
I assume there is not a lot of change in the docs-content for stuff that only landed in `angular/components#master`, but might be worth double-checking, or just using the `10.0.x` branch.